### PR TITLE
ref(TS) Convert some components to TS

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

File: react/features/base/tracks/functions.any.ts [hunk 1]
```diff
@@ -52,7 +53,7 @@ export function isParticipantMediaMuted(participant: IParticipant, mediaType: Me
  * @param {IReduxState} state - Global state.
  * @returns {boolean} - Is audio muted for the participant.
  */
-export function isParticipantAudioMuted(participant: IParticipant, state: IReduxState) {
+export function isParticipantAudioMuted(participant: IParticipant | undefined, state: IReduxState) {
```
Comment (horymury) [inline react/features/base/tracks/functions.any.ts:56]: can `participant` be undefined? optional params go to the end, if we got undefined maybe there s some bug?

File: react/features/filmstrip/components/web/AudioMutedIndicator.tsx [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
  */
-type Props = {
+interface IProps {
 
     /**
      * From which side of the indicator the tooltip should appear from.
      */
-    tooltipPosition: string
-};
+    tooltipPosition: 'top' | 'bottom' | 'left' | 'right';
```
Comment (horymury) [inline react/features/filmstrip/components/web/AudioMutedIndicator.tsx:14]: nit: maybe extract in a type POSITION = 'top' | 'bottom' | 'left' | 'right' and then assign props as type `tooltipPosition : POSITION` as I see several places where this is duplicated

File: react/features/filmstrip/components/web/ThumbnailAudioIndicator.tsx [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
 const JitsiTrackEvents = JitsiMeetJS.events.track;
 
-type Props = {
+interface IProps {
 
     /**
      * The audio track related to the participant.
      */
-    _audioTrack: ?Object
+    _audioTrack?: any;
```
Comment (horymury) [inline react/features/filmstrip/components/web/ThumbnailAudioIndicator.tsx:15]: isn't this an `ITrack`?

File: react/features/filmstrip/components/web/VirtualScreenshareParticipant.tsx [hunk 4]
```diff
@@ -1,102 +1,101 @@
...
      * The type of thumbnail.
      */
-    thumbnailType: string,
+    thumbnailType: string;
 
     /**
      * JitsiTrack instance.
      */
-    videoTrack: Object
+    videoTrack: any;
```
Comment (horymury) [inline react/features/filmstrip/components/web/VirtualScreenshareParticipant.tsx:98]: `Itrack` ?

File: react/features/participants-pane/components/web/MeetingParticipantItem.tsx [hunk 5]
```diff
@@ -30,152 +30,152 @@ import ParticipantActionEllipsis from './ParticipantActionEllipsis';
...
      * Media state for audio.
      */
-    _audioMediaState: MediaState,
+    _audioMediaState: MediaState;
 
     /**
      * The audio track related to the participant.
      */
-    _audioTrack: ?Object,
+    _audioTrack?: any;
```
Comment (horymury) [inline react/features/participants-pane/components/web/MeetingParticipantItem.tsx:43]: same

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/base/tracks/functions.any.ts (hunk 1 above)
Comment (robertpin) [reply react/features/base/tracks/functions.any.ts:56]: I'll change it

**Review by horymury — DISMISSED** [review]

**Review by horymury — APPROVED** [review]

**Pull request merged**